lib/index.js

```diff
@@ -68,6 +68,17 @@ function Server(opts) {
   app.handleMessage = function(providerId, data) {
     if (data && data.updates) {
       app.deltaCount++
+
+      var info = app.deltaCountPerProvider[providerId]
+      if (_.isUndefined(info)) {
+        info = {
+          count: 0,
+          percent: 0
+        }
+        app.deltaCountPerProvider[providerId] = info
+      }
+      info.count = info.count + 1
```

Comment:
++

Reply:
I’m also showing info about delta’s coming from plugins. Anything calling handleMessage. Going to get complicated if I have to figure out where to store the data.